testctrl/svc/types/event.go

```diff
@@ -41,9 +41,15 @@ type Event struct {
 	// Time is the point in time when the event was noticed.
 	Time time.Time
 
-	// Message is additional, unstructured information in a string. For example, it may describe
+	// Description is additional, unstructured information in a string. For example, it may describe
 	// inputs and and the error message with a ErrorEvent.
-	Message string
+	Description string
+
+	// DriverLogs contains a slice of bytes with the logs of the driver component. These logs
+	// contain a legible form of the results.
+	//
+	// This field will be nil unless Kind is a ErrorEvent, InternalErrorEvent, or DoneEvent).
+	DriverLogs []byte
```

Comment:
The `[]byte` was chosen over `string`, because the logs can technically be any encoding, are unstructured and the services don't use them. This is the way kubernetes returns them, and it is probably the correct type for this kind of field. 

The disadvantage is that a request made over the `grpc_cli`  will likely render them as binary in the response. That said, our services should be invoked with a legitimate client, eventually our CLI. I'd be open to `string` in the interim, if you prefer that for readability with `grpc_cli`.